homework13/src/main/java/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class CoffeeOrderBoard {
+    private final List<Order> orders = new ArrayList<>();
+    private int orderNumber = 1;
+
+    public void addOrder(Order order) {
```

Comment:
you dont need Order type as parameter, change to `String name` and life will be easier.